Source/Charts/Renderers/LineChartRenderer.swift

```diff
@@ -246,6 +246,10 @@ open class LineChartRenderer: LineRadarRenderer
         context.beginPath()
         context.addPath(cubicPath)
         context.setStrokeColor(drawingColor.cgColor)
+        context.setLineCap(CGLineCap.butt)
+        context.setLineJoin(CGLineJoin.round)
```

Comment:
If configuring line cap and line join, miter limit, flatness will improve the performance, please create a new var for each of them, and maybe help to test the performance on your side.

I'm not sure if it has to be hardcoded params.